x-pack/plugins/ml/public/application/components/job_selector/job_selector_flyout.tsx

```diff
@@ -51,6 +51,8 @@ export interface JobSelectorFlyoutProps {
   timeseriesOnly: boolean;
   maps: JobSelectionMaps;
   withTimeRangeSelector?: boolean;
+  applyTimeRangeConfig?: boolean;
+  setApplyTimeRangeConfig?: (v: boolean) => void;
```

Comment:
for controlled components the naming convention for callbacks is usually something like this 
```suggestion
  onTimeRangeConfigChange?: (v: boolean) => void;
```
providing a setter-like name imply a certain effect, but actually, the callback can do anything based on this change.

Reply:
Updated here [`82b3c33` (#97407)](https://github.com/elastic/kibana/pull/97407/commits/82b3c330d69d78c0386ca7164db80736b36ffe0b)